Mage.Tests/src/test/java/org/mage/test/cards/triggers/RadCounterTriggerTest.java

```diff
@@ -0,0 +1,109 @@
+package org.mage.test.cards.single.pip;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import mage.players.Player;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * > 725.1. Rad counters are a kind of counter a player can have (see rule 122, "Counters").
+ * There is an inherent triggered ability associated with rad counters. This ability has no
+ * source and is controlled by the active player. This is an exception to rule 113.8. The full
+ * text of this ability is "At the beginning of each player's precombat main phase, if that
+ * player has one or more rad counters, that player mills a number of cards equal to the number
+ * of rad counters they have. For each nonland card milled this way, that player loses 1 life
+ * and removes one rad counter from themselves."
+ *
+ * @author Susucr
+ */
+public class RadCounterTriggerTest extends CardTestPlayerBase {
```

Comment:
From started todo:
> Rad counters count needs to be added to player info panel

1. If you use fake emblem then no need to show player icon -- it can be shown as emblem in command zone.
2. Rad related cards must have card hint like "You have rad counters: xxx"

Reply:
I don't want to show Rad counter every game, so no I don't want to show it in the command zone.
The Rad trigger should be enabled always according to the rules. There might be shorcut on first moment a rad counter is added, but that is more complex to setup.